bin/node/runtime/src/lib.rs

```diff
@@ -222,13 +236,22 @@ impl InstanceFilter<Call> for ProxyType {
 			ProxyType::Staking => matches!(c, Call::Staking(..)),
 		}
 	}
+	fn is_superset(&self, o: &Self) -> bool {
+		match (self, o) {
+			(x, y) if x == y => true,
+			(ProxyType::Any, _) => true,
+			(_, ProxyType::Any) => false,
+			(ProxyType::NonTransfer, _) => true,
```

Comment:
Yeah I dont get this line. This is saying that if there exists a filter that is `NonTransfer`, and we want to check if it is a superset of anything, we return true?

Reply:
if the two filters are equal, or if the first is `Any`, then the first is a superset of the second. otherwise, if the second is `Any`, then the first is definitely not a superset of the second.

otherwise (if they're not equal and neither is `Any`), then the first is only a superset if it's `NotTransfer`.

Reply:
My misunderstanding was that this check is not a general one, but specific to the runtime and set of filters defined in it. So NonTransfer is specifically a super set when compared to governance and staking. all good